src/OVAL/probes/probe/main.c

```diff
@@ -257,6 +258,7 @@ int main(int argc, char *argv[])
 			if (chdir(rootdir) != 0) {
 				fail(errno, "chdir", __LINE__ -1);
 			}
+			dlopen("libgcc_s.so.1", RTLD_NOW | RTLD_GLOBAL);
```

Comment:
I don't think this is very portable. We should be detecting the path as part of `./configure` instead of using a hard-coded path. Do all other platforms we support have this library with the same name? What if you compile with `clang`?